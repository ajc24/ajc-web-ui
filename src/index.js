/* Export all of the header components */
export { default as Header } from './ui-elements/header/Header';

/* Export all of the icon components */
export { default as SmallDownArrowIcon } from './ui-elements/icons/SmallDownArrowIcon';
export { default as SmallRightArrowIcon } from './ui-elements/icons/SmallRightArrowIcon';
