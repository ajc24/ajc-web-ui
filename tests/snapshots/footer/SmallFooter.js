import 'jsdom-global/register';
import React from 'react';
import { TestDev } from 'ajc-jest-enzyme';
import { SmallFooter } from '../../../src';

describe('SmallFooter', () => {
  describe('Default props and rendering - Component with grey background', () => {
    let jsonSnapshot;

    beforeAll(() => {
      jsonSnapshot = TestDev.createSnapshot(
        <React.Fragment>
          <SmallFooter>
            This is a test.
          </SmallFooter>
        </React.Fragment>
      );
    });
    it('verifies the snapshot for the component', () => {
      expect(jsonSnapshot).toMatchSnapshot();
    });
  });

  describe('Transferred props and rendering - Component with red background', () => {
    let jsonSnapshot;

    beforeAll(() => {
      jsonSnapshot = TestDev.createSnapshot(
        <React.Fragment>
          <SmallFooter colour="red">
            This is a test.
          </SmallFooter>
        </React.Fragment>
      );
    });
    it('verifies the snapshot for the component', () => {
      expect(jsonSnapshot).toMatchSnapshot();
    });
  });
});
