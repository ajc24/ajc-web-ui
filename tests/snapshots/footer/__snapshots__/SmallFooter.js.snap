// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallFooter Default props and rendering - Component with grey background verifies the snapshot for the component 1`] = `
<div
  className="ajc-flex-row ajc-flex-justify-center ajc-footer-small ajc-width-4k ajc-background-grey-2"
  role="contentinfo"
>
  <div
    className="ajc-flex-row ajc-flex-justify-center ajc-width-full-hd"
  >
    <div
      className="ajc-flex-row ajc-flex-justify-center ajc-width-hd"
    >
      <div
        className="ajc-flex-row ajc-flex-justify-center ajc-footer-small-content ajc-width-hd-with-padding ajc-text-white ajc-font-default"
      >
        This is a test.
      </div>
    </div>
  </div>
</div>
`;

exports[`SmallFooter Transferred props and rendering - Component with red background verifies the snapshot for the component 1`] = `
<div
  className="ajc-flex-row ajc-flex-justify-center ajc-footer-small ajc-width-4k ajc-background-red"
  role="contentinfo"
>
  <div
    className="ajc-flex-row ajc-flex-justify-center ajc-width-full-hd"
  >
    <div
      className="ajc-flex-row ajc-flex-justify-center ajc-width-hd"
    >
      <div
        className="ajc-flex-row ajc-flex-justify-center ajc-footer-small-content ajc-width-hd-with-padding ajc-text-white ajc-font-default"
      >
        This is a test.
      </div>
    </div>
  </div>
</div>
`;
